src/Components/Articles.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { useState } from "react";
+import { useEffect } from "react";
+import { getArticles } from "../Utility";
+
+function Articles() {
+  const [articles, setArticles] = useState([]);
```

Comment:
We want to have a loading state here so that a message is briefly displayed whilst the data is still  loading from the api. :)